c/makeotf/makeotf_lib/source/hotconv/feattoks.h

```diff
@@ -1,6 +1,6 @@
 #ifndef feattoks_h
 #define feattoks_h
-/* feattoks.h -- List of labeled tokens and stuff
+/* feattoks.h -- List of labelled tokens and stuff
```

Comment:
Well, ok, but the American spelling is also fine.

Reply:
Huh. I think that was antlr’s i18n, not me, but I’ll drop it from the commit.